sql/core/src/main/scala/org/apache/spark/sql/execution/Window.scala

```diff
@@ -920,7 +931,7 @@ private[execution] object AggregateProcessor {
     // Create the projections.
     val initialProjection = newMutableProjection(
       initialValues,
-      Seq(SizeBasedWindowFunction.n))()
+      Seq(partitionSize))()
```

Comment:
Yeah the initial buffer will have more values. A `partitionSize` attribute will only be created/initialized when a `SizeBasedWindowFunction` is used. The `partitionSize` will be put at the first position, and others will be placed after that.

The initialization logic can be found here: https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/Window.scala#L965-L967